Advanced-Recruit-Training/src/BatteryStateOfChargeService.h

```diff
@@ -25,4 +26,6 @@ class BatteryStateOfChargeService : public I_BatteryStateOfChargeService
     const double initialStateOfChargePercent_;
 
     // Add your necessary variables here
+    BatteryData lastPoint;
```

Comment:
Point can mean alot of things, I would use `lastDataPoint_`, I think reading it with that is more fluent. 

Also, all private data members on the team should have a trailing underscore.